src/classes/InventoryManager.ts

```diff
@@ -39,18 +40,32 @@ export default class InventoryManager {
         }
 
         let genericCheck = generics;
+        let genericIndex = -1; // index of a generic match
         // if we looking at amount we can trade and the sku is a generic unusual, always set generic to true
         const isGenericSku = /^[0-9]*;5$/.test(sku);
         if (isGenericSku) {
             genericCheck = true;
         }
 
+        const gSku = SKU.fromString(sku);
+        gSku.effect = null;
+        if (generics) {
+            // figure out if we even have a generic sku
+            // Index of of item in Pricelist
+            genericIndex = this.pricelist.getIndex(null, gSku);
```

Comment:
cont.
Because if got both generic and non-generic in pricelist, this will always > -1

Reply:
amount can buy, generic max still overrides normal.I think you are right. I have made it such that it checks for both a generic and a normal sku. If it finds the normal sku, even in generic mode, it will use the normal sku for price. For